### ISSUE TITLE
NTR: ageusia/hypogeusia/dysgeusia

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5813]
see MP ticket: https://github.com/obophenotype/mammalian-phenotype-ontology/issues/3211
Do we want to add these terms in HPO too?

**Comment by pnrobinson** [comment]
Dysgeusia is probably parageusia.=, check!
https://hpo.jax.org/app/browse/term/HP:0000223

**Comment by nicolevasilevsky** [comment]
yes, Dysgeusia is parageusia, and it looks like it is already a synonym!

**Comment by pnrobinson** [comment]
All three of these items already existed. I changed the primary label of "Decreased taste sensation" to Hypogeusia, because "Decreased taste sensation" was already listed as a layperson synonym.

**Comment by nicolevasilevsky** [comment]
Ah, thanks @pnrobinson!
